sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -23,6 +27,10 @@
     _get_deserialize,
 )
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
you should be able to just import from typing_extensions (this check is done for you)